parser_with_temp.py

```diff
@@ -75,35 +75,37 @@ def get_top_number_of_general_page(self):
             re.search('\d+\)', summary_pages).group(0)[:-1])
 
     def get_id_list(self):
-        """Iterates a list up to last_page_number, calling pages with general
-        tables and grabs all order id's into a list with all orders id's
+        """Iterates a range up to last_page_number, calling pages with general
+        tables and grabs all order id's into a list.
         :return a list with all orders numbers.
         """
 
         _id_list = []
         for page_number in range(self.last_page_number):
+            print ("Getting list of id's. {} % completed".format(
+                page_number*100 / self.last_page_number))
             params = (
                 ('route', 'sale/order'),
                 ('page', page_number + 1),
-                ('token', self.token),
+                ('token', self.token)
             )
             table_page = BeautifulSoup(self.get(
                 self.base_url, params=params).text.encode(
                 'utf-8'), "html.parser")
-            table_body = table_page.find('table', {'class': 'list'}).find(
-                'tbody')
-            for row in table_body.findAll('tr'):
-                input_value = row.find('input').get('value')
-                if input_value:
-                    _id_list.append(int(input_value))
+            raw_inputs = table_page.select(
+                # 'table.list tbody tr:not(.filter) input')
```

Comment:
If this line is not needed, it may be removed at all.

Reply:
@pivancy removed